code/modules/hydroponics/trays/tray_process.dm

```diff
@@ -35,7 +35,10 @@
 		return
 
 	// Advance plant age.
-	if(prob(30)) age += 1 * HYDRO_SPEED_MULTIPLIER
+	current_cycle ++
```

Comment:
it's non-standard formatting
like literally the ++ operator is the only thing in this damn codebase that is always the same